src/atoms_calculators.jl

```diff
@@ -0,0 +1,69 @@
+import AtomsCalculators
+
+export ASEcalculator
+
+"""
+    ASEcalculator{T}
+
+This structure wraps python ASE calculator to AtomsCalculators
+interface compatible structure.
+
+# Example
+```julia
+using AtomsCalculators
+using ASEconvert
+using PythonCall
+
+fname = "path to eam potential file"
+EAM = pyimport("ase.calculators.eam")
+eam_cal = ASEconvert.ASEcalculator(EAM.EAM(potential=fname))
+
+atoms_ase = ase.build.bulk("Ni") * pytuple((4, 3, 2))
+atoms_ab = pyconvert(AbstractSystem, atoms_ase)
+
+AtomsCalculators.potential_energy(atoms_ab, eam_cal)
+AtomsCalculators.forces(atoms_ab, eam_cal)
+```
+
+"""
+mutable struct ASEcalculator{T}
+    ase_python_calculator::T
```

Comment:
Why mutable ?
Also does the typing help anything ?

Reply:
There are several mutable ASE calculators, so having mutable here is imo the best option.

Typing, I am not completely sure. But it is either `Any` or typing and I would say that typing is better option.

Reply:
I don't agree with that in general. Type variables in structs can e.g. make the stacktraces less readable. And here where in any case dynamic dispatch happens on the python level, I see no advantage of a type argument over Any. But I guess you can even type it. Should it not always be a `::Py` object (or whatever PythonCall uses as the type ... I recall it was Py).